src/UberStrok.WebServices/Core/BaseShopWebService.cs

```diff
@@ -170,13 +171,13 @@ byte[] IShopWebServiceContract.GetBundles(byte[] data)
             {
                 using (var bytes = new MemoryStream(data))
                 {                  
-                var channel = EnumProxy<ChannelType>.Deserialize(bytes);
+                    var channel = EnumProxy<ChannelType>.Deserialize(bytes);
 
-                using (var outBytes = new MemoryStream())
-                {
-                    EnumProxy<ChannelType>.Serialize(outBytes, channel);
-                    return outBytes.ToArray();
-                }
+                    using (var outBytes = new MemoryStream())
+                    {
+                        EnumProxy<ChannelType>.Serialize(outBytes, channel);

```

Comment:
The base class should not contain any hardcoded logic, instead rely on the abstract method's implementation.

```suggestion
List<BundleView> bundles = OnGetBundles(channel);
ListProxy<BundleView>.Serialize(outBytes, bundles, BundleView.Serialize)
```

Reply:
BundleView does not contain definition for Serialize

Reply:
Oops, apologies that should be `BundleViewProxy.Serialize`